### ISSUE TITLE
Lack of asynchronous DFFs

### ISSUE THREAD

**Issue by WenhMa** [bespoke-silicon-group/basejump_stl#339]
There are no asynchronous DFFs in Basejump STL. These cells are widely used in many cases. I think they are necessary for Basejump STL.

**Comment by dpetrisko** [comment]
Hello! To clarify, you mean flops with asynchronous reset?

**Comment by WenhMa** [comment]
Yes. All DFFs in Basejump Stl are synchronous. But, when I discuss the DFF type with many engineer, they thought that DFFs with asynchronous reset are prefered in many cases.

发件人: Dan Petrisko<mailto:notifications@github.com>
发送时间: 2020年11月23日 11:52
收件人: bespoke-silicon-group/basejump_stl<mailto:basejump_stl@noreply.github.com>
抄送: WenhMa<mailto:WenhMa@outlook.com>; Author<mailto:author@noreply.github.com>
主题: Re: [bespoke-silicon-group/basejump_stl] Lack of asynchronous DFFs (#339)


Hello! To clarify, you mean flops with asynchronous reset?

―

**Comment by taylor-bsg** [comment]
| 5 | Issue [#339](https://github.com/bespoke-silicon-group/basejump_stl/issues/339) — Lack of asynchronous DFFs | Close — implemented | bsg_dff_async_reset provides a parameterized vector DFF with asynchronous reset and configurable reset value. [current source](https://github.com/bespoke-silicon-group/basejump_stl/blob/11d19a888daa24e8239fc81c9c397b4d7bd58568/bsg_misc/bsg_dff_async_reset.sv#L3). |
